test/unit_test/instantiate_fft_tests.hpp

```diff
@@ -92,34 +92,34 @@ INSTANTIATE_TEST_SUITE_P(SubgroupTest, FFTTest,
 // and configurations
 INSTANTIATE_TEST_SUITE_P(SubgroupOrWorkgroupTest, FFTTest,
                          ::testing::ConvertGenerator<basic_param_tuple>(::testing::Combine(
-                             all_valid_placement_layouts, fwd_only, interleaved_storage, ::testing::Values(1, 131),
+                             all_valid_placement_layouts, fwd_only, complex_storages, ::testing::Values(1, 3, 131),
```

Comment:
is the batch size 3 catching a bug which was not evident with a larger batch size of 131 ?
If not. then could we do away with the batch size of 3, we already have a lot of tests as it is

Reply:
No, just easier to debug. I can remove it.

Reply:
I'd prefer that we remove it then. We're trying to reduce the number of tests

Reply:
done